[5.0.x] Updated translations from Transifex.

nessita opened the pull request.

felixxm commented:
@nessita I think it's too soon for this PR, we still have two weeks for translators and there is no way to update it easily later as many steps have to be done manually.

felixxm commented on django/conf/locale/af/LC_MESSAGES/django.po:
```diff
@@ -52,6 +52,9 @@ msgstr "Bosnies"
 
+msgid "Central Kurdish (Sorani)"
+msgstr ""
+
```
We avoid changing files without any new translations as this one.

felixxm commented on django/conf/locale/dsb/LC_MESSAGES/django.po:
```diff
@@ -6,10 +6,10 @@ msgid ""
-"POT-Creation-Date: 2023-01-17 02:13-0600\n"
-"PO-Revision-Date: 2023-04-25 06:49+0000\n"
+"POT-Creation-Date: 2023-09-18 11:41-0300\n"
+"PO-Revision-Date: 2013-04-25 06:49+0000\n"
```
`PO-Revision-Date` must be manually bumped to be later than `POT-Creation-Date`.

nessita replied:
On release day, will run:
```
find django -name '*.po' | xargs sed -ri "s/PO-Revision-Date: [0-9\-]+ /PO-Revision-Date: $(date -I) /g"
```

felixxm commented on django/conf/locale/fr/LC_MESSAGES/django.po:
```diff
@@ -13,15 +13,16 @@ msgid ""
 "Content-Transfer-Encoding: 8bit\n"
 "Language: fr\n"
-"Plural-Forms: nplurals=2; plural=(n > 1);\n"
+"Plural-Forms: nplurals=3; plural=(n == 0 || n == 1) ? 0 : n != 0 && n % "
```
Please revert changes in `Plural-Forms` for `fr` and `es*` languages.

felixxm commented on django/conf/locale/ar_DZ/LC_MESSAGES/django.po:
```diff
@@ -1369,9 +1384,9 @@ msgstr ""
-"URLs."
+"%(version)s/ref/settings/#debug\" target=\"_blank\" "
+"rel=\"noopener\">DEBUG=True</a> is in your settings file and you have not "
+"configured any URLs."
```
Please revert changes to all files without any new/changed translated strings (as this one).

nessita commented:
This is still is WIP!!! :stop_sign:

nessita commented:
@felixxm @claudep Hello! I believe this PR is ready for an initial review. I have:
1. fetched all translations from transifex
2. reverted those .po and .mo files without new translations
3. reverted plural-forms for es*, fr, pt* and it

I still need to sed-and-replace the `PO-Revision-Date` to be newer on changed files, I will do so Monday.

A few notes: there were new translations files/folders for `nl_BE`, `si` (only a few resources) and for `ug` (most resources). I have added those to the repo as well. Then, I had questions about what we do with langs ending in `Latin` or `Latn`, so we track those as well?

Then, after the 5.0 final release, I would like to talk a bit about this process and how we could improve it, but I'll reach out about this separately. Thanks!

nessita commented:
> > A few notes: there were new translations files/folders for `nl_BE`, `si` (only a few resources) and for `ug` (most resources). I have added those to the repo as well. Then, I had questions about what we do with langs ending in `Latin` or `Latn`, so we track those as well?
> 
> For adding a new language we require to at least fully translate the "core" resource on Transifex, then you have to prepare `formats.py` and make a few other changes including release notes (e.g. #12940, #12918, #13155, or #17163). That's not the case for `nl_BE`, `si`, `pap`, `tlh`, `lg`, please remove them from this patch.

Thanks, I did not know.

Removed the languages and made a separated commit, I will squash tomorrow morning before the release (along with the `PO-Revision-Date` changes).

felixxm commented:
@nessita Thanks :+1: I checked all the files and they look good (except `PO-Revision-Date`). I only have the following comments:
- I updated GIS translations for some languages to avoid missing translations caused by typographic changes, please refetch them:
  - `django/contrib/gis/locale/eo/LC_MESSAGES/django.[pm]o`
  - `django/contrib/gis/locale/es_CO/LC_MESSAGES/django.[pm]o`
  - `django/contrib/gis/locale/ia/LC_MESSAGES/django.[pm]o`
  - `django/contrib/gis/locale/lt/LC_MESSAGES/django.[pm]o`
  - `django/contrib/gis/locale/mk/LC_MESSAGES/django.[pm]o`
  - `django/contrib/gis/locale/mn/LC_MESSAGES/django.[pm]o`
  - `django/contrib/gis/locale/sl/LC_MESSAGES/django.[pm]o`
- I don't see any new translated strings in the `django/contrib/humanize/locale/zh_Hant/LC_MESSAGES/django.[pm]o`, please revert it.

nessita commented:
> > Then, I had questions about what we do with langs ending in Latin or Latn, so we track those as well?
> 
> I think so, however `ug@Latin` is practically empty. I'd remove it from this patch.

Sounds good, I removed it.
